### PR TITLE
Fix Baxter setup assistant roslaunch to work with xacro files

### PR DESCRIPTION
**Pull request by davetcoleman**

### CODE REVIEW

File: baxter/baxter_moveit_config/.setup_assistant [hunk 1]
```diff
@@ -0,0 +1,10 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: baxter_description
+    relative_path: urdf/baxter.urdf.xacro
+  SRDF:
+    relative_path: config/baxter.srdf
+  CONFIG:
+    author_name: Rethink Robotics Inc.
+    author_email: rsdk.support@rethinkrobotics.com
+    generated_timestamp: 1474511695
```
Comment (IanTheEngineer) [inline baxter/baxter_moveit_config/.setup_assistant:10]: I think you'll have a merge conflict with https://github.com/ros-planning/moveit_robots/pull/52 here.

File: baxter/baxter_moveit_config/README.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-PACKAGE DEPENDENCIES
----------
+## PACKAGE DEPENDENCIES
 To use the baxter_moveit_config package you will need the baxter_description package containing Baxter's URDF. This package is available for download at the following repository:
 
    git clone https://github.com/RethinkRobotics/baxter_common.git
 
+## Generate SRDF with XACRO
+
+To use the setup assistant, generate the latest baxter.srdf from the xacro file:
```
Comment (IanTheEngineer) [inline baxter/baxter_moveit_config/README.md:15]: The `SRDF` is generated at runtime through the `roslaunch` & `xacro` depending on your commandline arguments. I think its fine to tell users how to generate the `SRDF` for their own purposes, but what is the need to re-run setup assistant?

**Review by IanTheEngineer — COMMENTED** [review]
Is setup assistant something that needs to be again after the initial configuration? What exactly needs to be fixed?

File: baxter/baxter_moveit_config/.setup_assistant (hunk 1 above)
Comment (davetcoleman) [reply baxter/baxter_moveit_config/.setup_assistant:10]: i'll fix which ever one is merged second

File: baxter/baxter_moveit_config/README.md (hunk 2 above)
Comment (davetcoleman) [reply baxter/baxter_moveit_config/README.md:15]: For cases just like what I've run into - the URDF has changed for Baxter and the baxter_moveit_config has gotten out of date/sync with the newest changes in MoveIt!. Its also useful if you add a new link to your URDF and need to re-generate the collision matrix

File: baxter/baxter_moveit_config/.setup_assistant (hunk 1 above)
Comment (davetcoleman) [reply baxter/baxter_moveit_config/.setup_assistant:10]: instead i've just conslidated the PRs: https://github.com/ros-planning/moveit_robots/pull/59

**Comment by davetcoleman** [comment]
Superceded by https://github.com/ros-planning/moveit_robots/pull/59

**Pull request closed without merging**